sky/tools/create_full_ios_framework.py

```diff
@@ -164,8 +164,25 @@ def create_framework(
 
 
 def zip_archive(dst):
+  entitlement_config = open(dst + '/entitlements.txt', 'w')
+  entitlement_config.write('gen_snapshot_arm64\n')
+  entitlement_config.close()
+
+  without_entitlement_config = open(dst + '/without_entitlements.txt', 'w')
```

Comment:
This is repeated twice, maybe moving to a function and reusing here.

Reply:
Sure, modularized.